cmd/uplink/cmd/cp.go

```diff
@@ -34,6 +36,7 @@ func init() {
 		RunE:  copyMain,
 	}, RootCmd)
 	progress = cpCmd.Flags().Bool("progress", true, "if true, show progress")
+	expires = cpCmd.Flags().Int("expires", 0, "number of days before file deletion. Default (0) is indefinite")
```

Comment:
This probably should be specified as a `time.Duration` and such that you can specify `1yr`, `1day` etc.